haas/plugins/result_handler.py

```diff
@@ -217,3 +221,63 @@ def __call__(self, result):
             self.stream.write(" '{0}'".format(result.message))
         self.stream.writeln()
         self.stream.flush()
+
+
+class SlowTestsResultHandler(IResultHandlerPlugin):
+    separator1 = '=' * 70
```

Comment:
Class attributes make sense when it is static data shared with all instances. There is no reason to put it in the instance `__dict__`.